sklearn/utils/metaestimators.py

```diff
@@ -111,7 +111,10 @@ def __get__(self, obj, owner=None):
                 )
 
         # lambda, but not partial, allows help() to work with update_wrapper
-        out = lambda *args, **kwargs: self.fn(obj, *args, **kwargs)  # noqa
+        if obj is None:
```

Comment:
Maybe add a comment to state that this makes it possible to use the decorated method as an unbound method, for instance when monkeypatching.

Reply:
Got it!